CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release notes
 
+## Unreleased
+
+- Update to webpack5
+- Update a bunch of packages (most notably d3 packages which have a slightly different API)
+- Fix tests
+- Update scss to fix deprecation warnings
+- Remove pixi.js, react and react-dom as external packages
```

Comment:
I am confused here. 9 days ago or so, externals were added back to `webpack.config.js`. Is this an outdated bullet point?

Reply:
No, just didn't update the CHANGELOG to reflect that change. Good catch!